charts/worker/values.yaml

```diff
@@ -63,7 +63,7 @@ bootConfig:
 ManagementPlaneConfig:
   mongoConfig:
     enabled: true
-    mongodbUrl: "mongodb://admin:pa$$w0rd@mongo-datastore-svc.skyramp:27017/?authSource=admin&authMechanism=SCRAM-SHA-256"
+    mongodbUrl: "mongodb://admin:pa$$word@mongo-datastore-svc.skyramp:27017/?authSource=admin&authMechanism=SCRAM-SHA-256"
```

Comment:
Random question, but why was this required?

Reply:
This is required by the skyramp worker to push test results and active mocks to mongo db

Reply:
Oh what I mean is where was the password change done to require this?